### PR TITLE
Add Crop Componenet implemented

### PR DESCRIPTION
**Pull request by mahieakhtar**
Initial implementation for Add Crop. https://github.com/HackYourFuture-CPH/FP-class11/issues/3

### CODE REVIEW

**Comment by gizemcandemir** [comment]
hey Mahie, just wanted to remind that we need to rename all the file and folder names with kebab-case. If you already renamed it in local, remember that github doesn't track the case changes in the file/folder names, so you need to change the folder name to something else first, then commit, and then rename it again to original name with kebab-case and commit again, following with push this time.

File: src/client/components/Crop/Input/InputText.story.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (gizemcandemir) [reply src/client/components/Crop/Input/InputText.story.js]: file names should be also in kebab-case, right?

**Comment by mahieakhtar** [comment]
> hey Mahie, just wanted to remind that we need to rename all the file and folder names with kebab-case. If you already renamed it in local, remember that github doesn't track the case changes in the file/folder names, so you need to change the folder name to something else first, then commit, and then rename it again to original name with kebab-case and commit again, following with push this time.

**Comment by dpfernandes** [comment]
Add Crop was renamed to Add Batch https://github.com/HackYourFuture-CPH/FP-class11/pull/113

**Pull request closed without merging**